myproject_sql/views/views.py

```diff
@@ -0,0 +1,65 @@
+from pyramid.view import view_config
+from pyramid.response import Response
+from pyramid.httpexceptions import HTTPNotFound, HTTPFound
+from sqlalchemy.exc import DBAPIError
+from datetime import datetime
+
+from ..models import Entry
+import os
+
+HERE = os.path.dirname(__file__)
+TIME_FORMAT = '%Y-%m-%d %I:%M:%S'
+
+
+@view_config(route_name='home', renderer='../templates/index.jinja2')
+def my_view(request):
+    try:
+        query = request.dbsession.query(Entry)
+        list_of_entries = query.order_by(Entry.creation_date.desc()).all()
+    except DBAPIError:
+        return Response(db_err_msg, content_type='text/plain', status=500)
```

Comment:
I would question whether this is a good idea.  It's likely to hide information from us that we want when we are developing the site (it prevents us from seeing the traceback that might lead us to the specific error) and it leaks potentially sensitive data about our database in production.  

I'd let the DBAPIError raise without handling it.  The framework will issue a 500 error in production and in development you'll get a useful traceback.